core/src/main/java/bio/terra/pearl/core/dao/BaseJdbiDao.java

```diff
@@ -242,6 +242,34 @@ protected Optional<T> findWithChild(UUID id, String childIdPropertyName, String
         );
     }
 
+    protected List<T> findAllByPropertyWithChildren(String columnName, Object columnValue, String childIdPropertyName,
```

Comment:
this is cool

Reply:
concur, this will be very useful.  Can you add a comment clarifying what this does and the performance considerations?